src/azure-cli-core/azure/cli/core/commands/parameters.py

```diff
@@ -113,6 +119,14 @@ def _type(value):
     return params
 
 
+def enum_default(resource_type, enum_name, enum_val_name):
+    mod = get_versioned_models(resource_type, enum_name)
+    try:
+        return getattr(mod, enum_val_name).value
+    except AttributeError:
+        return None
```

Comment:
Hmm yes could do that.
Does this work?
`logger.debug('Unable to get param default %s.%s for %s', enum_name, enum_val_name, resource_type)`
In logs, it would look like:
`Unable to get param default Kind.storage for ResourceType.MGMT_STORAGE`